docs/AssignmentEntity.html

```diff
@@ -784,101 +784,8 @@ <h5>Returns:</h5>
 
     
 
-    
```

Comment:
Did you manually remove this or generated the docs and this is what it removed?

Reply:
I removed it manually. I'm not sure how these are generated, there was no documentation for them in the README 😢

Reply:
Hmm so running that command seems to regen the docs but it updates >20 files so I think we can do this is a separate PR? Really feel like this should just be a step in the build...

Reply:
Yeah if it hasn't been updated in a while it can be large. Just wasn't sure if there was any links that point to what you've deleted. Probably not though.